go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -466,6 +466,229 @@ func TestSystemTableQueries(t *testing.T) {
 	enginetest.RunQueryTests(t, newDoltHarness(t), BrokenSystemTableQueries)
 }
 
+func TestUnscopedDoltDiffSystemTable(t *testing.T) {
+	// There's a bug in queries with where clauses that compare column equality with a
+	// variable. These tests use "commit_hash in (@Commit1)" to work around that bug.
+	// https://github.com/dolthub/go-mysql-server/issues/790
+
+	var tests = []enginetest.ScriptTest{
+		{
+			Name: "unscoped-diff-system-table: basic case",
```

Comment:
No need to do name scoping like this (like we do in bats), because golang tests have built in nested scope via the t.Run methods. So just call them a simple name. Like this one would be something like "three tables"

Reply:
✅